gui-libs/wlroots/wlroots-9999.ebuild

```diff
@@ -18,7 +18,7 @@ fi
 
 LICENSE="MIT"
 SLOT="0/9999"
-IUSE="elogind icccm systemd x11-backend X"
+IUSE="elogind icccm seatd systemd x11-backend X"
 REQUIRED_USE="?? ( elogind systemd )"
```

Comment:
Does seatd change this requirement...?

Reply:
No because seatd doesn't conflicts with elogind/systemd, it's an additional method.

Reply:
You've lost me though. Surely seatd should satisfy the REQUIRED_USE? So that you can use wlroots without elogind or systemd?

Reply:
Yeah, that's the point but seatd isn't a logind implementation. It either provides it itself via it's own API or proxies it to a logind implementation.

Reply:
Right, I've got you - thanks for explaining